components/Voters/VotersList.tsx

```
@@ -0,0 +1,170 @@
+  const [expandedVoter, setExpandedVoter] = useState<string | null>(null)
+  const [showModal, setShowModal] = useState(false)
+  const [voters, setVoters] = useState<Voter[]>([])
+
+  const {
+    data: votersData,
+    isLoading,
+    error
+  } = usePollVotes(pollId)
+
+  useEffect(() => {
+    if (votersData && votersData.votes) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)